use_cases/eluc/experiments/predictor_experiments.ipynb

```diff
@@ -32,7 +32,7 @@
     "from persistence.serializers.sklearn_serializer import SKLearnSerializer\n",
     "from predictors.predictor import Predictor\n",
     "from predictors.neural_network.neural_net_predictor import NeuralNetPredictor\n",
-    "from predictors.sklearn.sklearn_predictor import LinearRegressionPredictor, RandomForestPredictor"
+    "from predictors.sklearn_predictor.sklearn_predictor import LinearRegressionPredictor, RandomForestPredictor"
```

Comment:
Had to rename this folder because it was getting confused with the real sklearn library. It took me an hour to debug this!!